### PR TITLE
feat(core,schemas): log sign-in

### PR DESCRIPTION
**Pull request by IceHe**
## Summary

Log sign-in

## Linear Issue Reference

LOG-2119

## Testing

Pass all existing tests.

image

### CODE REVIEW

File: packages/core/src/routes/session.ts [hunk 1]
```diff
@@ -122,16 +138,23 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
...
       assertThat(
         await hasUserWithEmail(email),
         new RequestError({ code: 'user.email_not_exists', status: 422 })
       );
 
       const passcode = await createPasscode(jti, PasscodeType.SignIn, { email });
+      ctx.log.passcode = passcode;
+
       await sendPasscode(passcode);
+      // TODO: ctx.log.connectorId = connectorId; // LOG-2230
```
[the remark this answers is not in the document]
Comment (IceHe) [reply packages/core/src/routes/session.ts]: Removed. PTAL

**Review by wangsijie — COMMENTED** [review]
@darcyYe is more familar with the sign in process.

File: packages/core/src/routes/session.ts [hunk 2]
```diff
@@ -171,18 +198,28 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     }),
     async (ctx, next) => {
       const { connectorId, code, state, redirectUri } = ctx.guard.body;
+      ctx.log = {
+        ...ctx.log,
```
Comment (gao-sun) [inline packages/core/src/routes/session.ts:202]: the spreading is not aligned with others?

Comment (IceHe) [reply packages/core/src/routes/session.ts]: Updated. PTAL

**Review by gao-sun — COMMENTED** [review]
LGTM, but need other eng's approval

**Review by demonzoo — APPROVED** [review]

**Pull request merged**